Improve detection for cd/dvd-cops version string

HeroponRikiBestest opened the pull request:
This removes the content check, moves the NE checks to the NE check section, and adds a PE check for the single currently confirmed section. 

One thing I was wondering- what's with the "(Unconfirmed - Please report to us on Github)" parts? They seem pretty confirmed, even being in existing redump entries. Is it supposed to be a placeholder for a safedisc-style version table?

```
/home/bestest/cdCopsSamples/2002MM/BMM.exe: DVD-Cops (Unconfirmed - Please report to us on Github) 2.01
/home/bestest/cdCopsSamples/2003MM/bmm.exe: DVD-Cops (Unconfirmed - Please report to us on Github) 1,60
/home/bestest/cdCopsSamples/aHugEu/AgentHugo.exe: CD-Cops (Unconfirmed - Please report to us on Github) 1,08
/home/bestest/cdCopsSamples/bib/bib.dll: CD-Cops 1.95
/home/bestest/cdCopsSamples/mofamen/h3blade.exe: CD-Cops (Unconfirmed - Please report to us on Github) 1.72
```

Fixes https://github.com/SabreTools/BinaryObjectScanner/issues/367

HeroponRikiBestest commented:
Forgot to mention, but, so it'll get autoclosed, this would fix https://github.com/SabreTools/BinaryObjectScanner/issues/367

mnadareski commented on BinaryObjectScanner/Protection/CDDVDCops.cs:
```diff
@@ -104,13 +82,39 @@ public class CDDVDCops : IContentCheck, IExecutableCheck<NewExecutable>, IExecut
+                {
+                    0x43, 0x44, 0x2D, 0x43, 0x6F, 0x70, 0x73, 0x2C,
+                    0x20, 0x76, 0x65, 0x72, 0x2E, 0x20
+                }, GetVersion, "CD-Cops (Unconfirmed - Please report to us on Github)"),
```
If you found it in something, it's no longer unconfirmed. This applies to all of the checks.

HeroponRikiBestest commented:
Implemented all of those. I also noticed and added one last thing, a check for Codefree. Let me know if there are any issues with that.

For my own documentation:
```
/home/bestest/cdCopsSamples: CD/DVD-Cops, CD/DVD-Cops, CD-Cops, CD-Cops Codefree
/home/bestest/cdCopsSamples/2002MM/BMM.exe: DVD-Cops 2.01
/home/bestest/cdCopsSamples/2003MM/bmm.exe: DVD-Cops 1,60
/home/bestest/cdCopsSamples/aHugEu/AGENTHUG.QZ_: CD/DVD/WEB-Cops, CD/DVD-Cops, PEtite
/home/bestest/cdCopsSamples/aHugEu/AgentHugo.exe: CD-Cops 1,08
/home/bestest/cdCopsSamples/aHugEu/AGENTHUGO.W_X: CD/DVD-Cops, RenderWare
/home/bestest/cdCopsSamples/aHugEu/cdcode.key: CD-Cops Codefree
/home/bestest/cdCopsSamples/aHugEu/CDCOPS.DLL: CD-Cops, CD-Cops
/home/bestest/cdCopsSamples/bib/bib.dll: CD-Cops 1.95
/home/bestest/cdCopsSamples/flap/cdcode.key: CD-Cops Codefree
/home/bestest/cdCopsSamples/flap/CDCOPS.DLL: CD-Cops, CD-Cops
/home/bestest/cdCopsSamples/flap/FGP.exe: CD-Cops 1,31
/home/bestest/cdCopsSamples/flap/FGP.QZ_: CD/DVD/WEB-Cops, CD/DVD-Cops, PEtite
/home/bestest/cdCopsSamples/flap/FGP.W_X: CD/DVD-Cops
/home/bestest/cdCopsSamples/mofamen/h3blade.exe: CD-Cops 1.72
/home/bestest/cdCopsSamples/roboRum/cdcode.key: CD-Cops Codefree
/home/bestest/cdCopsSamples/roboRum/CDCOPS.DLL: CD-Cops, CD-Cops
/home/bestest/cdCopsSamples/roboRum/ROBORUMB.QZ_: CD/DVD/WEB-Cops, CD/DVD-Cops, PEtite
/home/bestest/cdCopsSamples/roboRum/RoboRumble.exe: UNI-Cops
/home/bestest/cdCopsSamples/roboRum/RoboRumble.W_X: CD/DVD-Cops, RenderWare
```
